spec/models/language_spec.rb

```diff
@@ -0,0 +1,28 @@
+=begin
+#FreeClimb API
+
+#FreeClimb is a cloud-based application programming interface (API) that puts the power of the Vail platform in your hands. FreeClimb simplifies the process of creating applications that can use a full range of telephony features without requiring specialized or on-site telephony equipment. Using the FreeClimb REST API to write applications is easy! You have the option to use the language of your choice or hit the API directly. Your application can execute a command by issuing a RESTful request to the FreeClimb API. The base URL to send HTTP requests to the FreeClimb REST API is: /apiserver. FreeClimb authenticates and processes your request.
+
+The version of the OpenAPI document: 1.0.0
+Contact: support@freeclimb.com
+Generated by: https://openapi-generator.tech
+OpenAPI Generator version: 5.4.0
+
+=end
+
+require 'spec_helper'
+require 'json'
+require 'date'
+
+# Unit tests for Freeclimb::Language
+# Automatically generated by openapi-generator (https://openapi-generator.tech)
+# Please update as you see appropriate
+describe Freeclimb::Language do
+  let(:instance) { Freeclimb::Language.new }
+
+  describe 'test an instance of Language' do
+    it 'should create an instance of Language' do
+      expect(instance).to be_instance_of(Freeclimb::Language)
```

Comment:
General comment for enum based models - can we have any tests around the enum key/values?

Reply:
This is done in latest commit